app/src/main/java/com/infomaniak/mail/data/cache/mailboxInfo/MailboxController.kt

```diff
@@ -99,19 +101,10 @@ class MailboxController @Inject constructor(
             val remoteMailboxesIds = remoteMailboxes.map { remoteMailbox ->
 
                 SentryLog.d(RealmDatabase.TAG, "Mailboxes: Get current data")
-                val localMailbox = getMailbox(userId, remoteMailbox.mailboxId, realm = this)
+                val previousLocalValues = getMailbox(userId, remoteMailbox.mailboxId, realm = this)?.local?.copyFromRealm()
```

Comment:
What's the reason that makes this copyFromRealm() required? How does it fix the bimi issue? Because I'm not sure I understand correctly how copyFromRealm() should properly be used

Reply:
We are replacing the `mailboxLocalValues` with another one that's coming from Realm.
We need to `copyFromRealm()` it if we want it to work.

Reply:
Yeah I figured, but how does it work? What does it bring to the table? Why does it work? What's missing if we don't use it? How does it fix the bimi issue?

Reply:
You need to copy the value from Realm to be able to put it back in Realm later.
If we don't, Realm crashes, that's all.